src/plugins/home/public/application/components/use_case_card.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { EuiIcon } from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+
+import { ContentManagementPluginStart } from '../../../../content_management/public';
+
+interface Config {
+  title: string;
+  description: string;
+  icon: string;
+}
+
+export const WORKSPACE_USE_CASES = {
+  observability: {
+    id: 'observability',
+    config: {
+      title: i18n.translate('workspace.usecase.observability.title', {
+        defaultMessage: 'Observability',
+      }),
+      description: i18n.translate('workspace.usecase.observability.description', {
+        defaultMessage: 'Gain visibility into your application and infrastructure',
+      }),
+      icon: 'wsObservability',
+    },
+  },
+  'security-analytics': {
+    id: 'security-analytics',
+    config: {
+      title: i18n.translate('workspace.usecase.security.analytics.title', {
+        defaultMessage: 'Security Analytics',
+      }),
+      description: i18n.translate('workspace.usecase.analytics.description', {
+        defaultMessage: 'Enhance your security posture with advanced analytics',
+      }),
+      icon: 'wsSecurityAnalytics',
+    },
+  },
+  search: {
+    id: 'search',
+    config: {
+      title: i18n.translate('workspace.usecase.search.title', {
+        defaultMessage: 'Search',
+      }),
+      description: i18n.translate('workspace.usecase.search.description', {
+        defaultMessage: 'Discover and query your data with ease',
+      }),
+      icon: 'wsSearch',
+    },
+  },
+};
+
+export const registerUseCaseCard = (
```

Comment:
Do we need to declare this file as `tsx` file? Seems that this file doesn't export any components or elements.